rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -168,61 +169,51 @@
-                this.cost = initialCost;
-            }
-
-            public long nextCost() {
-                return cost++;
-            }
-        }
-
-        final CostProvider costProvider = new CostProvider(50001L);
-        Arrays.stream(new CallTransaction.Function[]{
-            UPDATE_COLLECTIONS,
-            RECEIVE_HEADERS,
-            REGISTER_BTC_TRANSACTION,
-            RELEASE_BTC,
-            ADD_SIGNATURE,
-            GET_STATE_FOR_BTC_RELEASE_CLIENT,
-            GET_STATE_FOR_DEBUGGING,
-            GET_BTC_BLOCKCHAIN_BEST_CHAIN_HEIGHT,
-            GET_BTC_BLOCKCHAIN_BLOCK_LOCATOR,
-            GET_MINIMUM_LOCK_TX_VALUE,
-            IS_BTC_TX_HASH_ALREADY_PROCESSED,
-            GET_BTC_TX_HASH_PROCESSED_HEIGHT,
-            GET_FEDERATION_ADDRESS,
-            GET_FEDERATION_SIZE,
-            GET_FEDERATION_THRESHOLD,
-            GET_FEDERATOR_PUBLIC_KEY,
-            GET_FEDERATION_CREATION_TIME,
-            GET_FEDERATION_CREATION_BLOCK_NUMBER,
-            GET_RETIRING_FEDERATION_ADDRESS,
-            GET_RETIRING_FEDERATION_SIZE,
-            GET_RETIRING_FEDERATION_THRESHOLD,
-            GET_RETIRING_FEDERATOR_PUBLIC_KEY,
-            GET_RETIRING_FEDERATION_CREATION_TIME,
-            GET_RETIRING_FEDERATION_CREATION_BLOCK_NUMBER,
-            CREATE_FEDERATION,
-            ADD_FEDERATOR_PUBLIC_KEY,
-            COMMIT_FEDERATION,
-            ROLLBACK_FEDERATION,
-            GET_PENDING_FEDERATION_HASH,
-            GET_PENDING_FEDERATION_SIZE,
-            GET_PENDING_FEDERATOR_PUBLIC_KEY,
-            GET_LOCK_WHITELIST_SIZE,
-            GET_LOCK_WHITELIST_ADDRESS,
-            ADD_LOCK_WHITELIST_ADDRESS,
-            REMOVE_LOCK_WHITELIST_ADDRESS,
-            SET_LOCK_WHITELIST_DISABLE_BLOCK_DELAY,
-            GET_FEE_PER_KB,
-            VOTE_FEE_PER_KB
-        }).forEach((CallTransaction.Function func) -> {
+        Arrays.stream(new Object[]{
+            Pair.of(UPDATE_COLLECTIONS, 48000L),
+            Pair.of(RECEIVE_HEADERS, 22000L),
+            Pair.of(REGISTER_BTC_TRANSACTION, 22000L),
+            Pair.of(RELEASE_BTC, 23000L),
+            Pair.of(ADD_SIGNATURE, 70000L),
+            Pair.of(GET_STATE_FOR_BTC_RELEASE_CLIENT, 4000L),
+            Pair.of(GET_STATE_FOR_DEBUGGING, 3_000_000L),
+            Pair.of(GET_BTC_BLOCKCHAIN_BEST_CHAIN_HEIGHT, 19000L),
+            Pair.of(GET_BTC_BLOCKCHAIN_BLOCK_LOCATOR, 76000L),
+            Pair.of(GET_MINIMUM_LOCK_TX_VALUE, 2000L),
+            Pair.of(IS_BTC_TX_HASH_ALREADY_PROCESSED, 23000L),
+            Pair.of(GET_BTC_TX_HASH_PROCESSED_HEIGHT, 22000L),
+            Pair.of(GET_FEDERATION_ADDRESS, 11000L),
+            Pair.of(GET_FEDERATION_SIZE, 10000L),
+            Pair.of(GET_FEDERATION_THRESHOLD, 11000L),
+            Pair.of(GET_FEDERATOR_PUBLIC_KEY, 10000L),
+            Pair.of(GET_FEDERATION_CREATION_TIME, 10000L),
+            Pair.of(GET_FEDERATION_CREATION_BLOCK_NUMBER, 10000L),
```

Comment:
GET_FEDERATION_ADDRESS
GET_FEDERATION_SIZE
GET_FEDERATION_THRESHOLD
GET_FEDERATION_CREATION_TIME
GET_FEDERATION_CREATION_BLOCK_NUMBER

look a little bit expensive, I expected these methods to cost between 2k and 3k like GET_MINIMUM_LOCK_TX_VALUE or GET_RETIRING_FEDERATION_ADDRESS